mnefun/_report.py

```diff
@@ -263,12 +266,21 @@ def _report_raw_segments(report, raw, lowpass=None):
     fig.set(figheight=(fig.axes[0].get_yticks() != 0).sum(),
             figwidth=12)
     fig.subplots_adjust(0.025, 0.0, 1, 1, 0, 0)
-    _add_figs_to_section(report, fig, section, section, image_format='png')
+    _add_figs_to_section(report, fig, section, section, image_format=img_form)
     print('%5.1f sec' % ((time.time() - t0),))
 
 
+def _gen_psd_plot(raw, fmax, n_fft, ax):
+    if hasattr(mne.time_frequency.Spectrum, 'plot'):
+        plot = raw.compute_psd(fmax=fmax, n_fft=n_fft).plot(show=False,
+                                                            axes=ax)
```

Comment:
this won't work actually: if there is no `Spectrum` class at all in the MNE version, then you get `AttributeError, module 'mne.time_frequency' has no attribute 'Spectrum'`

```suggestion
    if hasattr(raw, 'compute_psd'):
        plot = raw.compute_psd(fmax=fmax, n_fft=n_fft).plot(show=False,
                                                            axes=ax)
```